Wheel build breaking on 0.6.4

**Describe the bug**
```
Run pypa/cibuildwheel@v2.1.1
Run pipx run --spec '/home/runner/work/_actions/pypa/cibuildwheel/v2.1.1' cibuildwheel . --output-dir wheelhouse 2>&1
creating virtual environment...
determining package name from '/home/runner/work/_actions/pypa/cibuildwheel/v2.1.1'...
creating virtual environment...
installing cibuildwheel from spec '/home/runner/work/_actions/pypa/cibuildwheel/v2.1.1'...
cibuildwheel: Could not find any of {setup.py, setup.cfg, pyproject.toml} at root of package
Error: Process completed with exit code 2.
```

Probably from the change of python root folder. 
**To Reproduce**
Steps to reproduce the behavior:
1. 
2. 

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Version**
Branch, commit, version?

--- a/.github/workflows/integrate.yml
+++ b/.github/workflows/integrate.yml
@@ -112,7 +112,8 @@ jobs:
 
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.1.1
-        working-directory: path/to/your/package
+        with:
+          package-dir: ./python
 
       - uses: actions/upload-artifact@v5
         with:

--- a/.github/workflows/integrate.yml
+++ b/.github/workflows/integrate.yml
@@ -101,7 +101,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-    # if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
     steps:
       - uses: actions/checkout@v5
 

--- a/.github/workflows/integrate.yml
+++ b/.github/workflows/integrate.yml
@@ -101,7 +101,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+    # if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
     steps:
       - uses: actions/checkout@v5
 

--- a/.github/workflows/integrate.yml
+++ b/.github/workflows/integrate.yml
@@ -112,6 +112,8 @@ jobs:
 
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.1.1
+        env:
+          CIBW_PROJECT_DIR: python
 
       - uses: actions/upload-artifact@v5
         with:

--- a/.github/workflows/integrate.yml
+++ b/.github/workflows/integrate.yml
@@ -112,8 +112,7 @@ jobs:
 
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.1.1
-        env:
-          CIBW_PROJECT_DIR: python
+        working-directory: path/to/your/package
 
       - uses: actions/upload-artifact@v5
         with:

--- a/HISTORY.md
+++ b/HISTORY.md
@@ -7,6 +7,7 @@
 - [cpp]: fix: Cannot convert from 'initializer list' to 'toppra::BoundaryCond' (#245)
 - Move Python library to its own subfolder to support building as ROS 2 package (#266)
 - Fix Python and C++ CI, disable ECOS solver in tests (#270)
+- Update path for building wheels in CI (#272)
 
 ## 0.6.2 (Sept 19 2023)
 

